ci-operator/jobs/openshift/origin/openshift-origin-release-3.11-periodics.yaml

```diff
@@ -10,7 +10,6 @@ periodics:
     org: openshift
     repo: origin
   labels:
-    ci.openshift.io/release-type: informing
```

Comment:
@GeoBK you removed this and now we don't have a 3.11 informing dashboard in testgrid anymore.

why weren't these jobs added to https://github.com/openshift/release/blob/master/core-services/testgrid-config-generator/_allow-list.yaml when the label was removed here?

i've created https://github.com/openshift/release/pull/21694 to re-add them.